Bellawei0/CS157A-section9-team7
# Bella v1

Code Review Request: 
1) Finished Insertion Functionalities for uploading doc and pic Note 
2) Created InsertDocNoteServlet, InsertPicNoteServlet, insertDoc.jsp, insertPic.jsp classes 
3) Modified NoteDao and implemented two functions: insertDocNote and insertPicNote
4) Modified datatype from MySQL 

How to Test Code: 
Update Database from file V2__Create_person_table.sql. 
When test code, go to note page - > Add New Note - > Note page with dropdown menu > 
  a) Choose Doc, sample data  for "Enter JSON"  is:      **["Apple", "Banana", "Pages"]** . It has to be in JSON format as input 
  b) Choose PIC, upload file's size can not large than **16MB**
 c) Then go to Workbench, check tables of note_meta, note_docu, note_picture,  they should have the same note_id since note_docu and note_picture are subclasses of note_meta, they have the sample primary key.
